### PR TITLE
MM-46824 : Allows for messages from Bot and Webhook to be forwarded

### PR DESCRIPTION
**Pull request by dezerb**
#### Summary
Forward message: messages from bots cannot be forwarded

#### Ticket Link
fixes https://mattermost.atlassian.net/browse/MM-46824

#### Screenshots
-->
|  before  |  after  |
|----|----|
| image  | image|

#### Release Note

-->
```release-note
Allows for messages from Bot and Webhook to be forwarded
```

### CODE REVIEW

**Review by michelengelen — APPROVED** [review]
lgtm

File: components/dot_menu/dot_menu.tsx [hunk 1]
```diff
@@ -464,9 +464,7 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             </span>
         );
 
-        const fromWebhook = this.props.post.props?.from_webhook === 'true';
```
Comment (M-ZubairAhmed) [inline components/dot_menu/dot_menu.tsx:467]: we have `isFromWebhook` "utils/post_utils.ts". May be we can even add `isFromBot` check in that utils too.

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: components/dot_menu/dot_menu.tsx (hunk 1 above)
Comment (dezerb) [reply components/dot_menu/dot_menu.tsx:467]: @M-ZubairAhmed done and I updated some use cases.

**Review by M-ZubairAhmed — APPROVED** [review]
🆗  It looks good to me. Thank you

**Review by koox00 — APPROVED** [review]

**Comment by michelengelen** [comment]
removing @jgilliam17 as she is on PTO. @saturninoabril feel free to re-assign as you see fit.

**Comment by michelengelen** [comment]
@jgilliam17 this only needs a brief test ... could you take over please?

**Pull request merged**